src/main/resources/hibernate.cfg.xml

```diff
@@ -5,9 +5,7 @@
     <session-factory>
         <property name="hibernate.dialect"> org.hibernate.dialect.PostgreSQL95Dialect</property>
         <property name="hibernate.connection.driver_class"> org.postgresql.Driver</property>
-        <property name="hibernate.connection.username">postgres</property>
-        <property name="hibernate.connection.password">12345</property>
-        <property name="hibernate.connection.url"> jdbc:postgresql://srv-captain--youquiz-db-db:5432/hibernatedb</property>
+        <property name="hibernate.connection.username">postgres</property> <property name="hibernate.connection.password">12345</property> <property name="hibernate.connection.url"> jdbc:postgresql://postgres:5432/hibernatedb</property>
```

Comment:
These are incorrect conficvalues. Please revert merge from branch BE/dockerCompose